docs-new/docs/graphql/core/deployment/logging.mdx

```diff
@@ -93,7 +92,7 @@ For example, setting `--log-level=warn`, will enable all warn and error
 level logs only. So even if you have enabled `query-log` it won't be
 printed as the level of `query-log` is `info`.
 
-See `log-types <log-types>` for more details on log-level of each
+See [log-types](/graphql/core/deployment/logging.mdx/) for more details on log-level of each
```

Comment:
this is actually linking to the same file (this) only.
so only hash param is enough.

```suggestion
See [log-types](#log-types) for more details on log-level of each
```